dashboard/app/models/school_info.rb

```diff
@@ -131,14 +132,32 @@ def sync_from_schools
   validate :validate_without_country
   validate :validate_zip
 
+  # Decides whether the school info is complete enough to stop bugging the
+  # teacher for additional information every week.  Different from complete
+  # record validation.
   def complete?
+    return false if country.nil?
+    return true unless usa?
+    return true if [
+      SCHOOL_TYPE_HOMESCHOOL,
+      SCHOOL_TYPE_AFTER_SCHOOL,
+      SCHOOL_TYPE_ORGANIZATION,
+      SCHOOL_STATE_OTHER
```

Comment:
Should this be SCHOOL_TYPE_OTHER (not SCHOOL_STATE_OTHER)?

Reply:
![facepalm](https://user-images.githubusercontent.com/1615761/36288084-d118e530-126d-11e8-874d-da763b87555d.gif)
Thanks.